modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -72,6 +72,15 @@
     '/etc/lightdm/session-cleanup':
       mode    => '0755',
       source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/session-cleanup';
+    '/opt/share/xsession/lightdm-gtk-greeter.po':
+      source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/lightdm-gtk-greeter.po';
+  }
+
+  # overwrite greeter strings with OCF ones
+  exec { 'lightdm-greeter-compile-po':
+      command => 'msgcat /opt/share/xsession/lightdm-gtk-greeter.po | msgfmt -o \
+                    /usr/share/locale/en_US/LC_MESSAGES/lightdm-gtk-greeter.mo -',
+      require => File['/opt/share/xsession/lightdm-gtk-greeter.po'];
```

Comment:
You may wish to `package {'gettext':;}` and add `depends => Package['gettext']` to guarantee msgcat and friends are installed when this exec runs. not sure how early those get installed when we bootstrap new desktops.